dias/config_loader.py

```diff
@@ -94,6 +94,12 @@ def __init__(self, config_path, limit_task=None):
             )
             raise AttributeError(msg)
 
+        # initialise all of the trackers
+        self.trackers = {}
+        for t in self.global_config["trackers"]:
+            self.trackers[t['name']] = Tracker(t["path"], t["db_file"])
```

Comment:
Nice.
I'm wondering if we should add some checks here in case some keys are not present.

Reply:
Why is this resolved?

Reply:
Because I added the check! You are looking at outdated code.